synapse/storage/databases/main/roommember.py

```diff
@@ -146,14 +146,7 @@ def _transact(txn: LoggingTransaction) -> int:
 
     @cached(max_entries=100000, iterable=True)
     async def get_users_in_room(self, room_id: str) -> List[str]:
-        """
-        Returns a list of users in the room sorted by longest in the room first
-        (aka. with the lowest depth). This is done to match the sort in
-        `get_current_hosts_in_room()` and so we can re-use the cache but it's
-        not horrible to have here either.
-
-        Uses `m.room.member`s in the room state at the current forward extremities to
-        determine which users are in the room.
```

Comment:
This paragraph is still true (unless you think it's too obvious to document).

Reply:
I don't think its really helpful TBH (it was removed as part of the reversion).